pyvotca/numerical_polarizability.py

```diff
@@ -0,0 +1,150 @@
+"""Numerical polarizability.
+
+API
+---
+.. autoclass:: NumericalPolarizability
+
+"""
+
+import numpy as np
+import os
+import itertools
+import h5py
+
+from .molecule import Molecule
+from .utils import BOHR2ANG
+from .xtp import DFTGWBSE
+
+__all__ = ["NumericalPolarizability"]
+
+
+class NumericalPolarizability:
+    """Compute the polarizability numerically using XTP."""
+    def __init__(self,
+                 xtp: DFTGWBSE,
+                 dE: float = 0.002,
+                 path_to_simulations: str = './polar_experiments/'):
+        self.xtp = xtp
+        self.dE = dE
+        self.path = path_to_simulations
+
+    def mkfldr(self, path: str):
```

Comment:
Do you want to raise an error whenever the directory already exists? what kind of errors are you trying to catch?
I think that you are looking for something like:
```py
os.makedirs(name, exist_ok=True)
```
or
```py
from pathlib import Path
Path(name).mkdir(parents=True, exist_ok=True)
```